website/data/docs-nav-data.json

```diff
@@ -86,17 +86,26 @@
         "title": "Overview",
         "path": "install-boundary"
       },
-      {
-        "title": "Recommended architecture",
-        "path": "install-boundary/recommended-architecture"
-      },
       {
         "title": "System requirements",
         "path": "install-boundary/system-requirements"
       },
       {
-        "title": "Fault tolerance recommendations",
-        "path": "install-boundary/fault-tolerance"
+        "title": "Architecture",
+        "routes": [
+          {
+            "title": "Recommended architecture",
+            "path": "install-boundary/architecture/recommended-architecture"
+          },
+          {
+            "title": "Fault tolerance recommendations",
+            "path": "install-boundary/architecture/fault-tolerance"
+          },
+          {
+            "title": "Configure high availability",
+            "path": "install-boundary/architecture/high-availability"
+          }
+        ]
```

Comment:
I had an idea to group these together. Curious to hear other opinions on this.

Reply:
I'm a fan of the Architecture section you're introducing here. I might want to check out other product docs to see how they handle this, but generally I'm a fan

Reply:
Yeah, I like this a lot. These topics make sense together thematically - architecture decisions you'd want to think about before you start the install process. Grouping them together makes that Install Boundary guide look a lot cleaner.

Consul and Terraform both have **Architecture** sections like this, and the Consul one is included in the upcoming reorg they have planned. So I think there is precedent for grouping content like this.

I do like **System requirements** in this section too. I just pushed an update to move it under **Architecture**, but if anyone disagrees on seeing it there, I'm happy to move it back.

Unrelated to this change, but I also renamed **Fault tolerance recommendations** to **Fault tolerance** and **Configure high availability** to **High availability** on the TOC. I think "recommendations" is implied, not sure it has to be on the nav and it's cleaner without it. And the HA topic doesn't really contain configuration steps - it's more like considerations.